### PR TITLE
Reduced duplicated names in contributor list

### LINKED ISSUE #1421: Removal of duplicate names in contributors list
There are some persons in the contributors list whose names appear twice. Normally in the form of their Full name on one side and their github usernames elsewhere in the list. To show a more accurate list of contributors, the duplicates should be tracked down and removed.

### PR DESCRIPTION
**Pull request by lindaokorie27**
# Issue Description

The names in the contributors list are duplicated making the list very long

Fixes #1421 

## Changes Made

I checked the list using email addresses to make the search term more unique than using names

* **Please check if the PR fulfills these requirements**

- [x] PR is descriptively titled and links the original issue above
- [ ] Before/after screenshots (if this is a layout change)
- [ ] Details of which platforms the change was tested on (if this is a browser-specific change)
- [ ] Context for what motivated the change (if this is a change to some content)

### CODE REVIEW

File: site/contributors.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The contributors to this site, extracted from the
 [Git log](https://github.com/ocaml/ocaml.org/commits/master), are:
 
 <ul id="contributors_list">
-((! cmd git log --format="%aN" | sort | uniq | awk '{print "<li>"$1,$2,$3"</li>"}' !))
+((! cmd git log --format="%aE %aN" | sort | uniq | awk '{print "<li>"$2,"</li>"}' !))
```
Comment (gs0510) [inline site/contributors.md]: This results in printing only the first name of folks which isn't what we want. You want to list only the contributors using their email as a key for finding unique contributors.

**Review by gs0510 — COMMENTED** [review]
Thanks for your PR @LindieK! Were you able to check if the changes you made resulted in less duplications on the contributors.html page? Can you add a screenshot? Thanks!

File: site/contributors.md (hunk 1 above)
Comment (lindaokorie27) [reply site/contributors.md]: Oh sorry. Let me try again.

Comment (lindaokorie27) [reply site/contributors.md]: @gs0510 I have worked on it now. It displays contributor's full names. It removes names attached to the same email address.  There are a lot of contributors though that have multiple email addresses.

Comment (gs0510) [reply site/contributors.md]: @LindieK you can look into how to find only the uniq names in the list so awk can print only them. :)

Comment (lindaokorie27) [reply site/contributors.md]: Thanks for idea. I'll look into it right away

Comment (lindaokorie27) [reply site/contributors.md]: I sorted it further after the awk statement. The only thing left causing duplications is that the email and name documented on some commits are the person's github username (different from the full name used in other commit messages) and the auto-generated email from github that has the `@users.noreply.gtihub.com` extension.

**Review by gs0510 — COMMENTED** [review]
Awesome, thanks! LGTM :tada:
